examples/manual-approve-with-azure-devops/README.md

```diff
@@ -60,6 +61,7 @@ We need to configure a variable group with the name `TerraformProdDeploy`.  It s
 * `SERVICE_CONNECTION_NAME` - name of the Azure DevOps service connection for Azure Resource Manager that was defined earlier.
 * `DATABRICKS_HOST` - URL of the Databricks workspace where resources will be deployed.
 * `DATABRICKS_TOKEN` - personal access token for the Databricks workspace (follow [documentation](https://docs.databricks.com/dev-tools/api/latest/authentication.html) for instructions on how to obtain it).  Please mark this variable as **secret** to avoid exposing its value.
+* `PULL_REQUEST_COMMENT_TOKEN` - azure token with Pull Requests Treads R&W permissions. This token is used to comment the pull request.
```

Comment:
it looks like that it's possible to use system token, like it's described here: https://stackoverflow.com/a/60049855/18627